client/src/app/main.js

```diff
@@ -1,4 +1,4 @@
-var LernhowTemplates = ['GlobalNavView', 'HomeView', 'SignupView', 'NotFoundView', 'LoginView'];
+var LernhowTemplates = ['GlobalNavView', 'HomeView', 'SignupView', 'NotFoundView', 'LoginView', 'GlobalNavViewAlt'];
```

Comment:
Change view names for GlobalNav to be more semantic (GlobalNav, GlobalNavAuthenticated)